src/components/chart/examples/chart-items-clickable.tsx

```diff
@@ -0,0 +1,31 @@
+import { ChartItem } from '@limetech/lime-elements';
```

Comment:
This shouldn't be a `.tsx` file. `tsx` is for files that output jsx. Basically, only files that contain a Stencil `@Component` need to be `.tsx`. Otherwise it's a `.ts` file.

Reply:
I squashed this fixup. If there are no more comments @adrianschmidt @DavidAkkerman we can merge this now. Note that the component is still `@beta`. 

My next plan for this component is to add the "Chart Legend" feature.

Reply:
I haven't reviewed this yet, but I'm looking at it now.